src/App.scss

```diff
@@ -0,0 +1,13 @@
+a,
```

Comment:
- [ ] As you've pointed out in your pull request description, the mobile view hasn't been implemented yet.  Since this is a requirement, I'll leave a comment here.  I'm sure it will be beautiful! 💪

Reply:
Hi @Meltrust, thanks for the reply. We'll implement all the suggested changes.